AjaxControlToolkit.SampleSite/Temp/README.txt

```diff
@@ -0,0 +1,2 @@
+This file is added to the "Temp" directory, because Git does not support empty directories.
```

Comment:
No need to apologize for Git. Just describe why the directory is there:

> Temp directory for AjaxFileUpload.

Omit Global.asax details, because we may switch to web.config